docs/content/Getting-Started/user-quickstart.md

```diff
@@ -0,0 +1,80 @@
+
+[![QuickStart test]({{config.repo_url}}/actions/workflows/docs-ecutable-qs.yml/badge.svg?branch={{config.ks_branch}})]({{config.repo_url}}/actions/workflows/docs-ecutable-qs.yml)&nbsp;&nbsp;&nbsp;
+
+
+!!! tip "Estimated time to complete this example:" 
+    ~4 minutes (after installing prerequisites)
+
+### How to deploy and use KubeStellar
+
+This guide is intended to show how to (1) quickly bring up a **KubeStellar** environment using helm, (2) install plugins with brew, (3) retrieve your kubeconfig, (4) install KubeStellar Syncers on 2 edge clusters, (5) and deploy an example workload to both edge clusters.
+
+## 0. Pre-reqs
+
+helm
+brew
+kubectl
+3 kind clusters 
+   1 KubeStellar core cluster
+   2 KubeStellar edge clusters (edge-cluster1, edge-cluster2)
+   
+## 1. Install KubeStellar
+
+helm install kubestellar --set EXTERNAL_HOSTNAME="my-long-app-name.aregion.some.cloud.com" --set EXTERNAL_PORT=1024
+
+-or-
+
+helm install kubestellar --set clusterType=OpenShift
+
+## 2. Install KubeStellar's Kubectl plugins
+
+brew install kubestellar-kubectl v0.X.0
+
+## 3. Access KubeStellar
+
+kubectl get secrets kubestellar -o 'go-template={{index .data "external.kubeconfig"}}' | base64 --decode > admin.kubeconfig
```

Comment:
This seems a rather fiddly thing to ask users to do. Given that the previous step delivered executables intended for users to use, why not use https://docs.kubestellar.io/main/Coding%20Milestones/PoC2023q1/commands/#fetch-kubeconfig-for-external-clients ?